source/guides/scheduler-gpu-optimization.md

```diff
@@ -0,0 +1,270 @@
+dask-operator   dask-kubernetes-operator-775b8bbbd5-zdrf7   1/1     Running   0          74s
+```
+
+## Configuring a RAPIDS `DaskCluster`
+
+To configure the `DaskCluster` resource to run RAPIDS you need to set a few things:
+
+- The container image must contain RAPIDS, the [official RAPIDS container images](/tools/rapids-docker) are a good choice for this.
+- The Dask workers must be configured with one or more NVIDIA GPU resources.
+- The worker command must be set to `dask-cuda-worker`.
+
+## Creating a RAPIDS `DaskCluster` using `kubectl`
+
+Here is an example resource manifest for launching a RAPIDS Dask cluster with the scheduler optimzation
+
+```yaml
+# rapids-dask-cluster.yaml
+apiVersion: kubernetes.dask.org/v1
+kind: DaskCluster
+metadata:
+  name: rapids-dask-cluster
+  labels:
+    dask.org/cluster-name: rapids-dask-cluster
+spec:
+  worker:
+    replicas: 2
+    spec:
+      containers:
+        - name: worker
+          image: "nvcr.io/nvidia/rapidsai/base:24.06-cuda11.8-py3.10"
+          imagePullPolicy: "IfNotPresent"
+          args:
+            - dask-cuda-worker
+            - --name
+            - $(DASK_WORKER_NAME)
+          resources:
+            limits:
+              nvidia.com/gpu: "1"
+  scheduler:
+    spec:
+      containers:
+        - name: scheduler
+          image: "nvcr.io/nvidia/rapidsai/base:24.06-cuda11.8-py3.10"
+          imagePullPolicy: "IfNotPresent"
+          env:
+          args:
+            - dask-scheduler
+          ports:
+            - name: tcp-comm
+              containerPort: 8786
+              protocol: TCP
+            - name: http-dashboard
+              containerPort: 8787
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              port: http-dashboard
+              path: /health
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              port: http-dashboard
+              path: /health
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          resources:
+            limits:
+              nvidia.com/gpu: "1"
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              preference:
+                matchExpressions:
+                  - key: dedicated
+                    operator: In
+                    values:
+                      - scheduler
+    service:
+      type: ClusterIP
+      selector:
+        dask.org/cluster-name: rapids-dask-cluster
+        dask.org/component: scheduler
+      ports:
+        - name: tcp-comm
+          protocol: TCP
+          port: 8786
+          targetPort: "tcp-comm"
+        - name: http-dashboard
+          protocol: TCP
+          port: 8787
+          targetPort: "http-dashboard"
+```
+
+You can create this cluster with `kubectl`.
+
+```console
+$ kubectl apply -f rapids-dask-cluster.yaml
+```
+
```

Comment:
Could you also add an example that uses the Python API?

You'll want to use `make_cluster_spec` and then add the affinity section to the dictionary.

https://kubernetes.dask.org/en/latest/operator_kubecluster.html#customising-your-cluster

Reply:
I added the section which specifies on how to use KubeCluster for the same.